prow/istio-presubmit.sh

```diff
@@ -15,7 +15,6 @@
 # limitations under the License.
```

Comment:
Can you add lib in this file name, such that we know it is does not do anything by itself.